doc/changes/devel/12382.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix bad channels not handled properly in :func:`mne.stc_near_sensors` as well as default ``surface`` handling for :class:`mne.VolSourceEstimate` inputs as ``src``, by `Alex Rockhill`_.
```

Comment:
Not just a bugfix because it's involves a deprecation cycle. So the more important naming and change I think would be `12382.apichange.rst` describing how behavior is being changed in `stc_near_sensors`. Then you can have a `12382.bugfix.rst` to describe the bug fix separately if you want (but less critical than calling out the API change via deprecation in the right place in the changelog)